src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.ts

```diff
@@ -73,7 +63,8 @@ export class CodebasesToolbarComponent implements OnInit {
       appliedFilters: [],
       resultsCount: this.resultsCount,
       selectedCount: 0,
-      totalCount: 0
+      totalCount: 0,
+      tooltipPlacement: 'left'
```

Comment:
This tooltipPlacement is for the filter only. Currently, it appears truncated while positioned on the left.

![screen shot 2017-11-20 at 3 37 18 pm](https://user-images.githubusercontent.com/17481322/33040628-a2d90392-ce09-11e7-883d-a7e8acdddba0.png)